split-llvm-extract.cpp

```diff
@@ -153,10 +193,22 @@ int main(int argc, char **argv) {
   std::string extractFilename = "temp.bc";
   std::string extractProgram = "llvm-extract";
 
+  // The exact llvm-extract binary needs to be passed as an env 
```

Comment:
I think this probably needs documenting at the top of the program because this is really a way of *overriding* the default `llvm-extract` I think.